nocasedict/_nocasedict.py

```diff
@@ -48,6 +48,10 @@
 
 __all__ = ['NocaseDict']
 
+# Used as default value for parameters to detect that it has not been
```

Comment:
should be parameter (singular) or they (plural)

Reply:
DONE